src/routes/question.js

```diff
@@ -1,39 +1,39 @@
 const express = require('express');
-const onlyAuthenticated = require('../middleware/onlyAuthenticated');
+// const onlyAuthenticated = require('../middleware/onlyAuthenticated');
 const questionController = require('../controllers/question');
 
 const router = express.Router();
 
 router.get('/', questionController.getAllQuestions);
 
-router.post('/', onlyAuthenticated(), questionController.addNewQuestion);
+router.post('/', questionController.addNewQuestion); // onlyAuthenticated()
```

Comment:
I think you forgot to retrieve the file to the default state. There are also some random comments in the controller/question file that need to be removed.

Reply:
done